packages/medusa/src/services/product-variant-inventory.ts

```diff
@@ -112,16 +112,18 @@ class ProductVariantInventoryService extends TransactionBaseService {
       locationIds = stockLocations.map((l) => l.id)
     }
 
+    const inventoryServiceTx = this.inventoryService_.withTransaction(
+      this.activeManager_
+    )
```

Comment:
question (non-blocking): what are we doing with transactions in these places - @carlos-r-l-rodrigues, @adrien2p?

Reply:
Not sure the transaction is needed just to confirm the inventory. It is only a read operation and if it throws it will be because the retrieve or the read query failed, in that case the main transaction will be reverted. Also, no write in that flow would impact the inventory at that specific moment right?

Reply:
My thinking was more concerning this being a transaction that involves the inventory module which can be isolated and therefore might not share a db connection with `ProductVariantInventoryService`.

Reply:
Sorry, forgot to mentioned what I wanting to mentioned as well 😂 in a case where a method is using different modules that can be isolated, it would required to create a new orchestration here to handle the steps and rollbacks

Reply:
Exactly, the TO could be applied only in case we have "mutations" involving multiple services.

Reply:
Thanks for clarifying! @pKorsholm can you update accordingly :)